weaver/cli.py

```diff
@@ -933,12 +933,13 @@ def _update_files(self, inputs, url=None):
                     continue
                 if href.startswith("file://"):
                     href = href[7:]
-                if "://" not in href and not os.path.isfile(href):
+                if "://" in href or not os.path.isfile(href):
                     LOGGER.warning(
-                        "Ignoring potential local file reference since it does not exist. "
+                        "Ignoring potential file reference since it probably is a remote reference or does not exist. "
```

Comment:
I think the original "potential local file" made more sense, since the point of this code is to work only on local files. 
When something erroneous, but expected as local, is detected, then the warning is emitted. 
When files are valid remote references, there should not be a warning message since this is the expected behavior (maybe a debug log could be added though).

Reply:
Hmm.. ok then, but in that case we should change it back to a separate case, as was my first implementation, as the case represented by an `href` input type is not caught elsewhere in this method (and looking at https://github.com/crim-ca/weaver/blob/fix-cli-href-input/weaver/cli.py#L1037-L1043 , execute will always pass the input values through `_update_files`

This specific fix was because, using an input file containing and `href` tag, it was still uploaded to the vault with `ERROR    [weaver.cli] Execute failed. Scheme not supported for local file reference.
` as a result (why I originally used the `data` tag here : https://gitlab.com/crim.ca/clients/cubewerx/esa-snap-app/-/merge_requests/2#note_1116638381)

Reply:
Ok. Indeed. A separate `if` would be more appropriate in that case.

Reply:
Done. Reused the check to make sure href is not a local file